### ISSUE TITLE
Direct3D8 support from DXVK is missing

### ISSUE THREAD

**Issue by imamhs** [lutris/lutris#5621]
### Bug description

D8VK project which provides Direct3D 8 support on top of Vulkan has merged with DXVK project. So currently DXVK provides Direct3D 8 support in addition to Direct3D 9, 10 and 11. When DXVK is selected in Lutris for a game it does not provide Direct3D 8 support from DXVK project, only Direct3D 9, 10 and 11 provided.

To be able to able run a game with Direct3D 8 support from DXVK I have to manually copy the d3d8.dll file to the game executable folder and set it native in winecfg.

Lutris already doing this for Direct3D 9, 10 and 11 but not for Direct3D 8 with DXVK.

Please add Direct3D 8 from DXVK just like Direct3D 9, 10 and 11 so that older games are run using DXVK when DXVK is selected in Lutris.

### How to Reproduce

1. Go to '....'
3. Do '....'
4. See '....'

### Expected behavior

Allow Lutris configure Direct3D 8 with DXVK just like Lutris already doing for Direct3D 9, 10 and 11 using DXVK when configured to use DXVK

### Log output

```shell
2024-09-28 05:31:14,641: The Battle.net source is unavailable because Google protobuf could not be loaded: Descriptors cannot be created directly.
If this call came from a _pb2.py file, your generated code is out of date and must be regenerated with protoc >= 3.19.0.
If you cannot immediately regenerate your protos, some other possible workarounds are:
 1. Downgrade the protobuf package to 3.20.x or lower.
 2. Set PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python (but this will use pure-Python parsing and will be much slower).

More information: https://developers.google.com/protocol-buffers/docs/news/2022-05-06#python-updates
INFO     2024-09-28 05:31:14,788 [application.do_command_line:479]:Starting Lutris 0.5.17
INFO     2024-09-28 05:31:14,888 [startup.run_all_checks:120]:AMD Radeon RX 6600 XT (1002:73ff 1043:05d3 amdgpu) Driver 24.1.6
DEBUG    2024-09-28 05:31:15,032 [xrandr.get_resolutions:91]:Retrieving resolution list
DEBUG    2024-09-28 05:31:15,037 [xrandr._get_vidmodes:18]:Retrieving 19 video modes from XrandR
DEBUG    2024-09-28 05:31:15,074 [path_cache._update_missing_games:114]:Checking for missing games
DEBUG    2024-09-28 05:31:15,100 [api.check_stale_runtime_versions:52]:Modified at Sat 28 Sep 2024 05:10:08, will update after Sat 28 Sep 2024 11:10:08
DEBUG    2024-09-28 05:31:15,140 [lutriswindow.create_runtime_updater_cb:1244]:Runtime up to date
DEBUG    2024-09-28 05:31:21,217 [xrandr.get_outputs:26]:Retrieving display outputs
DEBUG    2024-09-28 05:31:21,222 [xrandr._get_vidmodes:18]:Retrieving 19 video modes from XrandR
lutris-wrapper: Apache AH64: Air Assault 
Started initial process 23539 from gamemoderun /home/imam/.local/share/lutris/runners/wine/wine-ge-8-26-x86_64/bin/wine /run/media/imam/Portable ga/Windows games/Apache AH-64 Air Assault/Apache.exe
Start monitoring process.
gamemodeauto: 
fsync: up and running.
wine: Using setpriority to control niceness in the [-10,10] range
Initial process has exited (return code: 0)
wine client error:b0: wine client error:d0: wine client error:110: Monitored process exited.
Exit with return code 0
DEBUG    2024-09-28 05:31:39,245 [monitored_command.on_stop:228]:Process 23537 has terminated with code 0
DEBUG    2024-09-28 05:31:39,246 [game.beat:924]:Game thread stopped
WARNING  2024-09-28 05:31:39,246 [game.on_game_quit:962]:Game still running (state: running)
INFO     2024-09-28 05:31:39,247 [game.stop:935]:Stopping Apache AH64: Air Assault  (wine)
DEBUG    2024-09-28 05:31:39,247 [game.stop_game:891]:Apache AH64: Air Assault  (wine) has run for 18 seconds
DEBUG    2024-09-28 05:31:39,247 [application.on_game_stopped:794]:Removing 14 from running IDs
DEBUG    2024-09-28 05:31:39,259 [game.stop_game:903]:Playtime: 4 minutes
DEBUG    2024-09-28 05:31:39,259 [game.on_game_quit:980]:Apache AH64: Air Assault  stopped at Sat, 28 Sep 2024 05:31:39
DEBUG    2024-09-28 05:31:39,287 [path_cache.add_to_path_cache:47]:Adding Apache AH64: Air Assault  (wine) to path cache
INFO     2024-09-28 05:31:43,521 [application.do_shutdown:1051]:Shutting down Lutris
```

### System Information

```shell
Vulkan support: YES
Esync support: YES
Fsync support: YES
Wine installed: YES
Gamescope: YES
Mangohud: YES
Gamemode: YES
Steam: YES
In Flatpak: NO
[System]
OS: Manjaro Linux 24.0.7 Wynsdey
Arch: x86_64
Kernel: 6.1.105-1-MANJARO
Desktop: KDE
Display Server: x11
[CPU]
Vendor: AuthenticAMD
Model: AMD Ryzen 5 5600X 6-Core Processor
Physical cores: 6
Logical cores: 12
[Memory]
RAM: 15.5 GB
Swap: 16.0 GB
[Graphics]
Vendor: AMD
OpenGL Renderer: AMD Radeon RX 6600 XT (radeonsi, navi23, LLVM 18.1.8, DRM 3.49, 6.1.105-1-MANJARO)
OpenGL Version: 4.6 (Compatibility Profile) Mesa 24.1.6-arch1.1
OpenGL Core: 4.6 (Core Profile) Mesa 24.1.6-arch1.1
OpenGL ES: OpenGL ES 3.2 Mesa 24.1.6-arch1.1
Vulkan Version: 1.3.279
Vulkan Drivers: AMD Radeon RX 6600 XT (RADV NAVI23) (1.3.278)
```

**Comment by strycore** [comment]
That's a lot of words for just "please update DXVK"! 

I'll do it this weekend

**Comment by danieljohnson2** [comment]
You will need a new release for this to work. In master we do list 'd3d8.dll' as a managed DLL, but 0.5.17 does not have it yet.

**Comment by strycore** [comment]
Going to marked that as fixed since it will be in the 0.5.18 release
